leakpro/attacks/mia_attacks/lira.py

```diff
@@ -45,17 +45,24 @@ def _configure_attack(self:Self, configs: dict) -> None:
 
         self.shadow_models = []
         self.num_shadow_models = configs.get("num_shadow_models", 64)
+        self.exclude_logit_threshold = configs.get("exclude_logit_threshold", 1)
 
         self.online = configs.get("online", False)
-        self.fixed_variance = configs.get("fixed_variance", False)
+
+        self.memorization = configs.get("memorization", False)
+        self.memorization_threshold = configs.get("memorization_threshold", 0.5)
```

Comment:
Why do you set different default values here than in the actual memorization function?

Reply:
To have some default setup, such that we only need to have one flag for memorization. If a user wants to have more customization they can allways add those flags to the config file